themes/socialblue/components/03-molecules/navigation/navbar/navbar-secondary--sky.js

```diff
@@ -0,0 +1,65 @@
+(function ($) {
+
+  Drupal.behaviors.navbarSecondaryScrollable = {
```

Comment:
Please add a top level comment to this function to explain what it's used for so we still know this in 6 months time.

Additionally, please add comments throughout the function to make it easier for other people than the author to follow what's going on. Comments that document **why** something happens are more useful than **what** happens because the latter can be seen by reading the code.

Reply:
The comment was added to the `navbar-secondary--sky.js`